### ISSUE TITLE
Properties 4 Test: cannot set width in cloned theme

### ISSUE THREAD

**Issue by waynevicknair** [maqetta/maqetta#958]
Create a new theme by cloning "claro".
Create a new HTML file and drop a button on the canvas.
Attempt to change the width to 200px in the properties view with the cascade showing.
Get the "use app.css....." dialog.
Switch themes to your clone.
Attempt to change the width to 200px in the properties view with the cascade showing.
Get the "these are global changes....." dialog.

Issues:
1) click on "Don't show again" checkbox and screen flickers but box is not checked
2) click on OK button, dialog is dismissed but width property is not set to 200px, trying to set again yields same result.

testing on 5a3bea0d3880f9688062cf435d3f82126c726dab build on macwayne2 with Mac/Chrome

**Comment by JonFerraiolo** [comment]
Multiple comments on this.

First, we might have to defer all of this bug out of M3 due to running out of time.

Assigning to Brad to look at the worst part of this bug: It looks like there is a URL problem when attempting to modify a theme CSS file from the page editor (Properties palette). When I tried to reproduce this bug, I see this in the console, which obviously is wrong: there shouldn't be a project1 subfolder under themes/

```
GET http://localhost:55557/maqetta/user/maqettaUser/ws/workspace/project1/themes/a1/project1/lib/dojo/dojo/resources/dojo.css 404 (Not Found)  dojo.css
```

Regarding the "Don't show again" checkbox, maybe we can comment it out for M3 if that's easy and safe. I would be afraid of trying to fix it correctly at this late date. Going forward (M4), I'm thinking we should preserve the "Don't show again" checkbox when updating a (writable) theme CSS file, but don't show that option for the read-only CSS dialog.

Finally, I think there are bugs where the Properties palette isn't updating properly in response to theme switching. I saw cascade entries for "claro" even after I switched to a different theme. For M4, maybe we need to publish a "theme switched" event and have the properties palette update its views in response to that event.
